python/tk_desktop/systray.py

```diff
@@ -81,9 +81,6 @@ def deactivate(self, deactivated):
     def __init__(self, parent=None):
         QtGui.QMainWindow.__init__(self, parent)
 
-        if sys.platform == "darwin":
-            self.setAttribute(QtCore.Qt.WA_MacNoShadow)
```

Comment:
Why do we remove this?

Reply:
This has been removed from [Qt5](https://doc.qt.io/qt-5/macos-issues.html) to [Qt6](https://doc.qt.io/qt-6/macos-issues.html). Reference https://bugreports.qt.io/browse/QTBUG-94052?focusedCommentId=568642

Reply:
Description updated